_targets/build.project.armv7a9-zynq7000

```diff
@@ -85,6 +85,9 @@ b_build_target() {
 		"${PREFIX_PROG_STRIPPED}plo-${TARGET_FAMILY}-${TARGET_SUBFAMILY}.img" \
 		0
 
+	# Copy plo.elf with symbols to provide loading plo using gdb
+	cp "${PREFIX_PROG}plo-${TARGET_FAMILY}-${TARGET_SUBFAMILY}.elf" "$PREFIX_BOOT/"
```

Comment:
We can use the same convention as below, shorter names, instead of `plo-armv7a9-zynq7000.elf` use `plo-gdb.elf`.

Reply:
Thanks, I've applied suggested changes